servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/SubscribablePublisher.java

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
copied over. Can probably be inlined into the operator since it's not used anywhere else.